client/new-stack-editor/lib/views/flexsplit/resizer.coffee

```diff
@@ -0,0 +1,122 @@
+kd = require 'kd'
+FlexSplit = require './index'
+
+
+module.exports = class FlexSplitResizer extends kd.View
+
+  constructor: (options = {}, data) ->
+
+    options.type     ?= FlexSplit.HORIZONTAL
+    options.cssClass  = 'flex-resizer'
+    options.draggable = { axis: options.type.axis }
+
+    super options, data
+
+    @views      = []
+    @_fractions = []
+    { @type, view } = @getOptions()
+
+    @addView view
+
+    @on 'DragFinished', @dragFinished
+    @on 'DragStarted',  @dragStarted
+
+
+  _getViewIndex: (view) ->
+    if view is @views[0] then 0 else 1
+
+
+  addView: (view) ->
+
+    @views.push view
+
+    view.on FlexSplit.EVENT_EXPAND, =>
+
+      return  if view.hasClass 'expanded'
+      view.setClass 'expanded'
```

Comment:
Can we depend on something else other than a `css class`?

I am guessing this might be avoidable via putting this check into `view` class itself.

So this could be rewritten as:

```coffee
return  if view.isExpanded()
view.expand()
```

We will have the advantage of not changing anything if we want to change the way it's represented as `expanded` by only calling `isExpanded` and `expand` (presumably with `isCollapsed` and `collapse` helpers as well) on the view instance.